src/js/controller/model.js

```diff
@@ -170,39 +182,44 @@ define([
                 newItem = index;
             }
 
-            if (repeat || newItem !== _this.item) {
-                _this.item = newItem;
-                _this.sendEvent(events.JWPLAYER_PLAYLIST_ITEM, {
-                    index: _this.item
-                });
+            if (newItem === this.item && !repeat) {
+                return;
+            }
 
-                // select provider based on item source (video, youtube...)
-                var item = _this.playlist[newItem];
-                var source = item && item.sources && item.sources[0];
-                if (source === undefined) {
-                    // source is undefined when resetting index with empty playlist
-                    return;
-                }
-                var Provider = _providers.choose(source);
-                if (!Provider) {
-                    throw new Error('No suitable provider found');
-                }
+            this.item = newItem;
+            this.trigger(events.JWPLAYER_PLAYLIST_ITEM, {
+                index: this.item
+            });
 
-                // If we are changing video providers
-                if (!(_currentProvider instanceof Provider)) {
-                    _currentProvider = new Provider(_this.id);
+            // select provider based on item source (video, youtube...)
+            var item = this.playlist[newItem];
+            var source = item && item.sources && item.sources[0];
+            if (source === undefined) {
+                // source is undefined when resetting index with empty playlist
+                return;
+            }
 
-                    _this.setVideoProvider(_currentProvider);
-                }
+            var Provider = _providers.choose(source);
+            if (!Provider) {
+                throw new Error('No suitable provider found');
+            }
 
-                // this allows the Youtube provider to load preview images
-                if (_currentProvider.init) {
-                    _currentProvider.init(item);
-                }
+            // If we are changing video providers
+            if (!(_currentProvider instanceof Provider)) {
+                _currentProvider = new Provider(_this.id);
+
+                _this.setVideoProvider(_currentProvider);
+            }
+
+            // this allows the Youtube provider to load preview images
+            if (_currentProvider.init) {
+                _currentProvider.init(item);
             }
+
+            this._qoeItem = new Timer();
```

Comment:
this should match the item property above (might return/throw before we get here)